src/Serilog.Sinks.AzureDataExplorer.Samples/Properties/launchSettings.json

```diff
@@ -0,0 +1,15 @@
+{
+  "profiles": {
+    "Serilog.Sinks.AzureDataExplorer.Samples": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "ingestionURI": "https://ingest-bentest20230502.eastus.kusto.windows.net",
+        "databaseName": "logs",
+        "tableName": "Serilogs",
+        "appId": "5cb7e86c-6247-45ab-9939-db9621126106",
+        "appKey": "-4M8Q~3uN7ureKzq1OBd4RkQUBB4C7AeByeNTaKK",
+        "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
You probably don't want to merge these into the source code.

Reply:
sorry, my bad. The file has been removed and the sp has been remove as well.